release/ray_release/cluster_manager/minimal.py

```diff
@@ -248,6 +251,39 @@ def create_cluster_compute(self, _repeat: bool = True):
                     f"ID {self.cluster_compute_id}"
                 )
 
+    @property
+    def cluster_compute_with_extra_tags(self) -> dict:
+        if self.extra_tags and self._get_cloud_provider() == "AWS":
+            cluster_compute = self.cluster_compute.copy()
+            if "aws" in cluster_compute:
+                cluster_compute["aws"] = deepcopy(cluster_compute["aws"])
+            else:
+                cluster_compute["aws"] = {}
+            cluster_compute["aws"].setdefault("TagSpecifications", [])
+            tag_specifications = cluster_compute["aws"]["TagSpecifications"]
```

Comment:
We can simplify this a bit

```suggestion
            cluster_compute = self.cluster_compute.copy()
            aws = deepcopy(self.cluster_compute.get("aws", {}))
            tag_specifications = cluster_compute["aws"].setdefault("TagSpecifications", [])
```